crates/rerun/Cargo.toml

```diff
@@ -20,13 +20,20 @@ all-features = true
 targets = ["x86_64-unknown-linux-gnu", "wasm32-unknown-unknown"]
 
 
+[[bin]]
+# For the `rerun` binary, always add the `native_viewer` feature.
+# See https://github.com/rerun-io/rerun/issues/1997
+name = "rerun"
+required-features = ["native_viewer"]
```

Comment:
Should we maybe also put the `web-viewer` for this one...? that would avoid the dreaded `cargo install rerun -F web_viewer`...

Reply:
I would also slow down our own `cargo r -p rerun` a lot 😭